content/get-started/writing-on-github/editing-and-sharing-content-with-gists/forking-and-cloning-gists.md

```diff
@@ -12,11 +12,6 @@ versions:
   ghae: '*'
   ghec: '*'
 ---
-## Forking gists
```

Comment:
I think it would be strange to have an article called "Forking and cloning gists" without mentioning forking at all. Can you clarify why you're removing this section?  I agree the screenshot itself is not particularly useful and we could definitely delete, but would suggest keeping the rest of the section.